<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UISettings</class>
 <widget class="QDialog" name="UISettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>856</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <item>
    <widget class="QTreeWidget" name="twMenu">
     <property name="minimumSize">
      <size>
       <width>210</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>180</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="rootIsDecorated">
      <bool>true</bool>
     </property>
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Properties</string>
      </property>
     </column>
     <item>
      <property name="text">
       <string>General</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Editor</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>REPL</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Lint</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QStackedWidget" name="swPages">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="currentIndex">
        <number>2</number>
       </property>
       <widget class="QWidget" name="pGeneral">
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QGroupBox" name="gbWorkspace">
           <property name="title">
            <string>Opened file list sorting mode</string>
           </property>
           <layout class="QGridLayout">
            <property name="margin">
             <number>5</number>
            </property>
            <property name="spacing">
             <number>3</number>
            </property>
            <item row="0" column="0">
             <widget class="QRadioButton" name="rbOpeningOrder">
              <property name="text">
               <string>Opening order</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QRadioButton" name="rbFileName">
              <property name="text">
               <string>File name</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QRadioButton" name="rbUri">
              <property name="text">
               <string>URI</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QRadioButton" name="rbSuffix">
              <property name="text">
               <string>Suffix</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="gbFilesToHide">
           <property name="title">
            <string>Ignored files</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>This files are not displayed by File Browser, Locator, ignored when searching in directory</string>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_4">
              <property name="font">
               <font>
                <italic>true</italic>
               </font>
              </property>
              <property name="text">
               <string>One glob per line</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPlainTextEdit" name="pteFilesToHide"/>
            </item>
            <item>
             <spacer name="verticalSpacer_3">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="pEditor">
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>547</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Select a page in the tree</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="pLint">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Linters check your code for common mistakes.&lt;br/&gt;Linting results are shown near line numbers after the file was opened or saved. Clear it with &lt;span style=&quot; font-style:italic;&quot;&gt;Esc&lt;/span&gt;.&lt;br/&gt;You can easily extend Enki with linter for your language, if it is not supported yet.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>435</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="pRepl">
        <layout class="QVBoxLayout" name="verticalLayout_14">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>REPL means Read-Eval-Print loop - interactive console which allows you to execute current file and type additional commands</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="dbbButtons">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../icons/enkiicons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>dbbButtons</sender>
   <signal>accepted()</signal>
   <receiver>UISettings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>296</x>
     <y>587</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dbbButtons</sender>
   <signal>rejected()</signal>
   <receiver>UISettings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>364</x>
     <y>587</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
